src/app/modules/convai-checker/convai-checker.component.ts

```diff
@@ -360,16 +360,16 @@ export class ConvaiCheckerComponent implements OnInit, OnChanges {
   }
 
   private _getErrorMessage(error: any): string {
-    let msg = 'Error scoring text. Please try again.';
+    let msg =
+      'We\'re sorry, we\'re having trouble at the moment. Please try again later.';
     // Look at detailed API error messages for more meaningful error to return.
     try {
-      for (const api_err of error.json().errors) {
+      for (const api_err of error.error.errors) {
```

Comment:
Nice! Does this fix the broken language error?

Reply:
Yep!